LICENSE

```diff
@@ -336,4 +336,4 @@ This General Public License does not permit incorporating your program into
 proprietary programs.  If your program is a subroutine library, you may
 consider it more useful to permit linking proprietary applications with the
 library.  If this is what you want to do, use the GNU Lesser General
-Public License instead of this License.
\ No newline at end of file
+Public License instead of this License.
```

Comment:
I don't understand what should have changed here. Do you have any idea?

Reply:
Nope.